tests/integration/helpers.py

```diff
@@ -119,3 +132,29 @@ async def get_memberships(ops_test: OpsTest, url):
     user_id = data["vXGroupUsers"][0].get("userId")
     membership = (group, user_id)
     return membership
+
+
+async def get_or_add_model(
```

Comment:
Is this being used anywhere, or just a remnant of testing the cross-controller relations?

Reply:
Ah no it is not, thank you. I will remove.